react/features/overlay/reducer.js

```diff
@@ -54,3 +58,17 @@ function _mediaPermissionPromptVisibilityChanged(
 function _suspendDetected(state) {
     return set(state, 'suspendDetected', true);
 }
+
+/**
+ * Reduces a specific redux action {@code SET_FATAL_ERROR} of the feature
+ * overlay.
+ *
+ * @param {Object} state - The redux state of the feature overlay.
+ * @param {Error} fatalError - If the value is set it indicates that a fatal
+ * error has occurred and that the reload screen is to be displayed.
+ * @returns {Object}
+ * @private
+ */
+function _setFatalError(state, { fatalError }) {
+    return set(state, 'fatalError', fatalError ? fatalError : undefined);
```

Comment:
minor nit: I don't think we need the ternary operator here, setting it to whatever the value is will do.

Reply:
No, it will not do. I don't want 'false' here. Either error or undefined.

Reply:
Or maybe whatever. There's no type checking anyway so someone can set there true anyway